### PR TITLE
Update varnish-configuration.rst

### PR DESCRIPTION
**Pull request by ilithium**
Corrected grammar in introductory paragraph

### CODE REVIEW

File: doc/varnish-configuration.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@ Varnish Configuration
 ---------------------
 
 Below you will find detailed Varnish configuration recommendations for the
-features provided by this library. The examples are tested with Varnish
+features provided by this library. These examples have been tested with Varnish
```
Comment (dbu) [inline doc/varnish-configuration.rst]: not sure, the examples actually *are* constantly tested on travis-ci... but this is too much detail here, maybe we can just say that we provide configuration for version 3 and 4, without talking about testing at all. wdyt?

Comment (ilithium) [inline doc/varnish-configuration.rst]: That's a fair point; perhaps the wording: "the examples below are provided for Varnish 3 and 4." - would that suit?

Comment (dbu) [inline doc/varnish-configuration.rst]: actually its not examples, but reference documentation. lets say:

... features provided by this library. The configuration is provided for Varnish 3 and 4.

Comment (ilithium) [inline doc/varnish-configuration.rst]: Sure, that works with me - thanks!

--
Justin Finkelstein
Technical Director
ilithium limited
http://uk.linkedin.com/in/justinfinkelstein/

**Comment by dbu** [comment]
thanks. pushed that change directly to master.

**Pull request closed without merging**